MatrixSDK/Crypto/Verification/MXKeyVerificationManager.m

```diff
@@ -63,13 +63,13 @@ @interface MXKeyVerificationManager ()
     dispatch_queue_t cryptoQueue;
 
     // All running transactions
-    MXUsersDevicesMap<MXKeyVerificationTransaction*> *transactions;
+    MXUsersDevicesMap<MXDefaultKeyVerificationTransaction *> *transactions;
```

Comment:
`MXKeyVerificationManager` is accessing some of the internal methods of request and transaction which are defined in `_Private.h` header files and not exposed in the protocol. For this reason it is still using implementation types rather than protocol types